latch/registry/table.py

```diff
@@ -541,6 +560,119 @@ def _add_record_deletes_selection(
 
         mutations.append(res)
 
+    # upsert column
+
+    def create_column(
+        self,
+        key: str,
+        type: Union[Type[RegistryPrimitivePythonValue], str, List[str]],
+        *,
+        allow_multiple_entries: bool = False,
+        required: bool = False,
+    ):
+        """Create a column.
+
+        A transport error is thrown if a column of the same name already exists.
+
+        Args:
+            name: New column name.
+            type: Type of new column. If a string is provided, it is treated
```

Comment:
```suggestion
            type: Type of the new column. If a string is provided, it is treated
```
+ update the rest (tho it should be self-describing based on the function type